assets/src/apps/delivery/layouts/deck/DeckLayoutFooter.tsx

```diff
@@ -444,6 +444,29 @@ const DeckLayoutFooter: React.FC = () => {
     }
   }, [lastCheckResults, isPreviewMode]);
 
+  const isActivityHistoryTimeStampIsZero = () => {
```

Comment:
the name of this function would imply the return type would be a boolean value, but it is more like `getCurrentActivityVisitTimestamp`